channels/us.m3u

```diff
@@ -706,7 +706,7 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
 https://cbsn-ny-cedexis.cbsnstream.cbsnews.com/out/v1/ec3897d58a9b45129a77d67aa247d136/master.m3u8
 #EXTINF:-1 tvg-id="CBSNNewYorkPlutoTV.us" tvg-name="CBSN New York (Pluto TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ubsNr5F.jpg" group-title="Local",CBSN New York (Pluto TV)
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5dc48170e280c80009a861ab/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=206&deviceId=5dc48170e280c80009a861ab&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
-#EXTINF:-1 tvg-id="CBSNewsBoston.us" tvg-name="CBSNews Boston" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/a2Qjf7T.png" group-title="Local",CBSNews Boston
+#EXTINF:-1 tvg-id="CBSNewsBoston.us" tvg-name="CBS News Boston" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/a2Qjf7T.png" group-title="Local",CBS News Boston
```

Comment:
I've seen you have did this. I have also began to rename the ones above accordingly, but didn't finish. Might be useful.